### PR TITLE
feat(hubble): support highlighting and hints in query statements 

### PR DESCRIPTION
**Pull request by wanganjuan**
Query statements support highlighting and hints

image

### CODE REVIEW

**Review by imbajin — APPROVED** [review]
is this PR finished?

**Comment by wanganjuan** [comment]
> is this PR finished?

finished

File: hugegraph-hubble/hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx [hunk 1]
```diff
@@ -197,6 +201,8 @@ export const GremlinQuery: React.FC = observer(() => {
       {
         lineNumbers: true,
         lineWrapping: true,
+        mode: { name: 'text/x-mysql' },
```
Comment (lionztt) [inline hugegraph-hubble/hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx:204]: It looks like is mysql's hint config. I think that we should set config as gremlin's hint.

Comment (imbajin) [reply hugegraph-hubble/hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx:204]: seems right catch, thanks for ur reminder, would u also like to improve the FE module?

Comment (lionztt) [reply hugegraph-hubble/hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx:204]: yes, I'm learning the FE module codes

Comment (imbajin) [reply hugegraph-hubble/hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx:204]: > yes, I'm learning the FE module codes

any update for this PR, it should consider merge and improve it later if no better solution now

**Comment by wanganjuan** [comment]
Add gremlin keyword hints, basic function
image

**Comment by wanganjuan** [comment]
> Add gremlin keyword hints, basic function image

@imbajin

**Review by simon824 — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Pull request merged**